src/App.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'react';
+import './App.css';
+import Modal from './Components/Modal';
+
+function App() {
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const appTitle = `Componente modal con React`
```

Comment:
Why this vars? 🤔

Reply:
The first is the useState hook that is used to send the state to Modal component and then we can change this state inside the Modal component.

The second is only to store the label of the title and then call it on the h1 tag.